### DISCUSSION TITLE
image in xcube

### DISCUSSION THREAD

**Discussion by luca-moscatelli** [dcs4cop/xcube#1065] [Q&A]
I i need a opinion, my goal is to link some images in propertyMapping of placegroup but the images need to be private, there is some way like s3 or other way to render private images on the viewer for the authenticated users ?

**Comment by TonioF** [comment]
PlaceGroups refer to vector data, and they are bound to one or more datasets. What sort of images to you want to add to the viewer? They would need to be georeferenced, so you can add them as datasets. In that case, you can also specify the audience, i.e., make them visible only for authenticated users.
